components/VcItemActivationStatus.tsx

```diff
@@ -13,9 +13,9 @@ const WalletUnverifiedIcon: React.FC = () => {
     <Icon
       name="shield-alert"
       color={Theme.Colors.Icon}
-      size={28}
+      size={22}
```

Comment:
same here also. probably getting from theme like ICON_MID_SIZE. will help us in making our icons sizes consistent

Reply:
moved to themes